wcfsetup/install/files/lib/system/io/HttpFactory.class.php

```diff
@@ -33,15 +34,16 @@ final class HttpFactory
      */
     public static function getDefaultUserAgent(?string $comment = null): string
     {
+        $domain = ApplicationCacheBuilder::getInstance()->getData()['application'][1]->domainName;
```

Comment:
Why are you using `ApplicationCacheBuilder` instead of `ApplicationHandler` (same below)?

Reply:
503 brain not available. I forgot it was called `ApplicationHandler` and searched for a cache-class; just fixed it.